srcs/main.c

```diff
@@ -48,8 +48,8 @@ int	main(int ac, char **av)
 	{
 		if (ac == 2)
 			get_next_line(fd, &line);
-		//else
-		//	line = readline(prompt); // prompt???
+		else
+			line = readline("> ");
```

Comment:
Huuuuuum. So I'm a bit confused by readline. Do you have an example on what it does ? I read the description in the man but I'm not getting very well why you would pass something (or, well, that) to it ^^ I imagined it worked like gnl directly from the terminal, but maybe not ?
Or maybe you used it as an example though, to see how it works, and my whole point is useless haha.

On a side note : For future reference, I tend to like making macros or variables instead of putting directly the string (that way the code reads, instead of having to be interpreted when you look at it. Does that make sense?  I think a lot in terms of "the code should read like  in a book, it should explain itself in the line"). I know it takes more space, but for reading clarity....
For that one though, I don't know I guess. Undecided on if I care a lot or not. It might also not be relevant depending on what your intention was here.

Reply:
Ya I also didnt get it before I saw an example aha, the prompt is literally the prompt, it's what it prints to the terminal while waiting for the input